src/wallet/rpcdump.cpp

```diff
@@ -1482,3 +1482,263 @@ UniValue importmulti(const JSONRPCRequest& mainRequest)
 
     return response;
 }
+
+static UniValue ProcessDescriptorImport(CWallet * const pwallet, const UniValue& data, const int64_t timestamp) EXCLUSIVE_LOCKS_REQUIRED(pwallet->cs_wallet)
+{
+    UniValue warnings(UniValue::VARR);
+    UniValue result(UniValue::VOBJ);
+
+    try {
+        if (!data.exists("desc")) {
+            throw JSONRPCError(RPC_INVALID_PARAMETER, "Descriptor not found.");
+        }
+
+        const std::string& descriptor = data["desc"].get_str();
+        const bool active = data.exists("active") ? data["active"].get_bool() : true;
```

Comment:
Maybe we should also have a parameter to indicate where to set the next index when we are setting a descriptor as active.

Reply:
Sounds good. Probably optional? Default could be the beginning of the range.

Reply:
Yeah, that makes sense.